### DISCUSSION TITLE
NEW Ux/UI design

### DISCUSSION THREAD

**Discussion by dunottrue** [Asim-Tahir/audio-channel-ext#2] [Ideas]
I create new design for you application.
Please use it.

https://www.figma.com/design/fTOkQfgoSAXdhOkgRWfFOm/WebEQ-Node.js-project?node-id=0-1&t=hUVp5x0zsd3LzJ4Q-1
my email: dunottrue@gmail.com
Screenshots:

![WebEQ Node js project](https://github.com/user-attachments/assets/54245d7e-9d87-4b3d-9004-1ff76384185e)

**Comment by Asim-Tahir** [comment]
Hi,

Thanks for the great designs. Its awesome.

For the clarification this browser extension not mine(i wish but not 🤷🏻). I am just one of the few people who did not update the extension when it was changed in the webstore by the extension owner.

I wont promise but i want to polish this extension with modern tech stack and migrate to mv3. When i try it, definitely use your design as base. Thanks again. I remain this discussion open for future.

**Reply by Asim-Tahir** [reply]
I request access from figma. Looks like its private @dunottrue

**Reply by dunottrue** [reply]
Доступ выдал. Дизайн если будете использовать, то нужно будет поправить для верстальщика код. Дизайн делал давно ( я тогда был в этом деле новичок) мог наделать ошибок.  используйте бесплатно как угодно мой дизайн, потому что всё равно он либо пропадёт, либо я удалю его со временем из портфолио. если будут вопросы, можете писать мне либо на почту dunottrue@gmail.com либо тут на github. Но тут я редко бываю, а почту читаю чаще.

**Comment by dunottrue** [comment]
By the way, you can use SVG instead of PNG throughout the entire project, so that it always looks high-quality on any display.
